UnitTests/GitCommandsTests/MockExecutable.cs

```diff
@@ -94,37 +95,46 @@ public IProcess Start(ArgumentString arguments, bool createWindow, bool redirect
 
         private sealed class MockProcess : IProcess
         {
-            public MockProcess([CanBeNull] string output, int? exitCode)
+            public MockProcess([CanBeNull] string output, int? exitCode = 0)
             {
                 StandardOutput = new StreamReader(new MemoryStream(Encoding.UTF8.GetBytes(output ?? "")));
                 StandardError = new StreamReader(new MemoryStream());
                 StandardInput = new StreamWriter(new MemoryStream());
-                ExitCode = exitCode;
+                _exitCode = exitCode;
             }
 
             public MockProcess()
             {
                 StandardOutput = new StreamReader(new MemoryStream());
                 StandardError = new StreamReader(new MemoryStream());
                 StandardInput = new StreamWriter(new MemoryStream());
+                _exitCode = 0;
             }
 
             public StreamWriter StandardInput { get; }
             public StreamReader StandardOutput { get; }
             public StreamReader StandardError { get; }
 
-            public int? ExitCode { get; }
+            private int? _exitCode { get; }
```

Comment:
please keep it `ExitCode` or turn into a field

Reply:
Corrected.